### PR TITLE
refactor: improve variable namings

### PR DESCRIPTION
**Pull request by ditoglez**

### CODE REVIEW

File: apps/idos-data-dashboard/src/core/idos/idos-provider.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const Provider = ({ children }: PropsWithChildren) => {
     if (profile) {
       // @ts-expect-error
       await _sdk.setSigner(signer.type, signer.value);
-      const _pk = _sdk.auth.currentUser.publicKey;
+      const _pk = _sdk.auth.currentUser.currentUserPublicKey;
```
Comment (pkoch) [inline apps/idos-data-dashboard/src/core/idos/idos-provider.tsx:65]: This is missing the key's purpose: it's it encryption or auth?

File: packages/issuer-sdk-js/README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ const idos = await idOS.init(...);
 const { humanId } = await yourServer.getIdosInformation();
 
 // Discover user encryption key
-const { encryptionPublicKey } = await idos.discoverEncryptionKey(humanId);
+const { encryptionPublicKey } = await idos.discoverUserEncryptionPublicKey(humanId);
```
Comment (pkoch) [inline packages/issuer-sdk-js/README.md:82]:
```suggestion
const { userEncryptionPublicKey } = await idos.discoverUserEncryptionPublicKey(humanId);
```

File: packages/idos-sdk-js/src/lib/idos.ts [hunk 3]
```diff
@@ -102,8 +101,8 @@ export class idOS {
     return assertNever(type, `Signer type "${type}" not recognized`);
   }
 
-  async hasProfile(address: string): Promise<boolean> {
-    return this.kwilWrapper.hasProfile(address);
+  async hasProfile(user: string): Promise<boolean> {
```
Comment (pkoch) [inline packages/idos-sdk-js/src/lib/idos.ts:104]: Don't you mean `userAddress`?

File: apps/idos-enclave/src/lib/enclave.js [hunk 4]
```diff
@@ -48,10 +48,10 @@ export class Enclave {
     });
   }
 
-  storage(humanId, signerAddress, signerPublicKey, expectedUserEncryptionPublicKey) {
+  storage(humanId, signerAddress, signerEncryptionPublicKey, expectedUserEncryptionPublicKey) {
```
Comment (pkoch) [inline apps/idos-enclave/src/lib/enclave.js:51]: `signerEncryptionPublicKey` doesn't make sense. A signer is a crypto device used for auth.

What am I missing?

File: examples/idos-example-dapp/api/EVM.ts [hunk 5]
```diff
@@ -19,7 +19,7 @@ const evmGranteeSigner = new ethers.Wallet(
 const idosGrantee = await idOSGrantee.init({
   chainType: "EVM",
   granteeSigner: evmGranteeSigner,
-  encryptionSecret: ENCRYPTION_SECRET_KEY,
+  encryptionPrivateKey: ENCRYPTION_SECRET_KEY,
```
Comment (pkoch) [inline examples/idos-example-dapp/api/EVM.ts:22]: Unsure if you wanna do this now or later:
```suggestion
  recipientEncryptionPrivateKey: ENCRYPTION_SECRET_KEY,
```

File: packages/idos-sdk-js/src/lib/auth.ts [hunk 6]
```diff
@@ -14,8 +14,15 @@ import { implicitAddressFromPublicKey } from "./utils";
 
 export interface AuthUser {
   humanId: string | null;
```
Comment (pkoch) [inline packages/idos-sdk-js/src/lib/auth.ts:16]: Pretty sure this should be `user`. No?

File: packages/idos-sdk-js/src/lib/data.ts [hunk 7]
```diff
@@ -291,7 +292,7 @@ export class Data {
   async share(
     tableName: string,
     recordId: string,
-    receiverPublicKey: string,
+    granteeEncryptionPublicKey: string,
```
Comment (pkoch) [inline packages/idos-sdk-js/src/lib/data.ts:295]:
```suggestion
    granteeRecipientEncryptionPublicKey: string,
```

File: packages/idos-sdk-js/src/lib/enclave-providers/iframe-enclave.ts [hunk 8]
```diff
@@ -2,7 +2,7 @@ import type { idOSCredential } from "@idos-network/idos-sdk-types";
 import * as Base64Codec from "@stablelib/base64";
 import type { BackupPasswordInfo } from "../types";
 import type {
-  DiscoverEncryptionKeyResponse,
+  DiscoverUserEncryptionPublicKeyResponse,
```
Comment (pkoch) [inline packages/idos-sdk-js/src/lib/enclave-providers/iframe-enclave.ts:5]:
```suggestion
  DiscoverUserRecipientEncryptionPublicKeyResponse,
```

File: packages/idos-sdk-server-dapp/src/idOS-grantee.ts [hunk 9]
```diff
@@ -86,7 +86,7 @@ const buildKwilSignerAndGrantee = (
 };
 
 interface idOSGranteeInitParams {
-  encryptionSecret: string;
+  encryptionPrivateKey: string;
```
Comment (pkoch) [inline packages/idos-sdk-server-dapp/src/idOS-grantee.ts:89]:
```suggestion
  recepientEncryptionPrivateKey: string;
```

**Review by pkoch — COMMENTED** [review]
There are a bunch of things that need to be corrected. Only made a note on the first occurence I spotted.

File: apps/idos-enclave/src/lib/enclave.js (hunk 4 above)
Comment (Mohammed-Mamoun98) [reply apps/idos-enclave/src/lib/enclave.js]: ``userEncryptionPublicKey`` ?

**Comment by pkoch** [comment]
Superseded by https://github.com/idos-network/idos-sdk-js/pull/396

**Pull request closed without merging**